PendingReleaseNotes.md

```diff
@@ -29,3 +29,5 @@ v1.6...
 * Multiple Ceph mgr daemons are supported for stretch clusters and other clusters where HA of the mgr is more critical
 * Ceph OSD: as of Nautilus 14.2.14 and Octopus 15.2.9 if the OSD scenario is simple (one OSD per disk) we won't use LVM to prepare the disk anymore
 * Disable CSI GRPC metrics by default
+* Configure all Rook pods that were using the default service account (mon,mds,etc) to instead use a new `rook-ceph-default` service account.
```

Comment:
Suggest rewording here to remove "Configure all" which users might interpret as an instruction. 

Suggestion:
Ceph daemon pods using the `default` service account now use a new `rook-ceph-default` service account.